core/pom.xml

```diff
@@ -30,6 +43,18 @@
             <artifactId>hibernate-validator</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why did you add these dependencies? Most of the tests are failing when running mvn test on my side :

```
Running com.dooapp.fxform.reflection.classcast.ClassCastTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.002 sec <<< FAILURE!
classCastTest(com.dooapp.fxform.reflection.classcast.ClassCastTest)  Time elapsed: 0.002 sec  <<< ERROR!
java.lang.NoSuchMethodError: 'javax.el.ExpressionFactory javax.el.ELUtil.getExpressionFactory()'
	at javax.el.ELManager.getExpressionFactory(ELManager.java:38)
	at org.hibernate.validator.messageinterpolation.ResourceBundleMessageInterpolator.buildExpressionFactory(ResourceBundleMessageInterpolator.java:171)
	at org.hibernate.validator.messageinterpolation.ResourceBundleMessageInterpolator.<init>(ResourceBundleMessageInterpolator.java:94)
	at org.hibernate.validator.internal.engine.AbstractConfigurationImpl.getDefaultMessageInterpolator(AbstractConfigurationImpl.java:573)
	at org.hibernate.validator.internal.engine.AbstractConfigurationImpl.getDefaultMessageInterpolatorConfiguredWithClassLoader(AbstractConfigurationImpl.java:822)
	at org.hibernate.validator.internal.engine.AbstractConfigurationImpl.getMessageInterpolator(AbstractConfigurationImpl.java:483)
	at org.hibernate.validator.internal.engine.ValidatorFactoryImpl.<init>(ValidatorFactoryImpl.java:154)
	at org.hibernate.validator.HibernateValidator.buildValidatorFactory(HibernateValidator.java:38)
	at org.hibernate.validator.internal.engine.AbstractConfigurationImpl.buildValidatorFactory(AbstractConfigurationImpl.java:451)
	at javax.validation.Validation.buildDefaultValidatorFactory(Validation.java:103)
	at com.dooapp.fxform.validation.DefaultFXFormValidator.createValidator(DefaultFXFormValidator.java:50)
	at com.dooapp.fxform.validation.DefaultFXFormValidator.<init>(DefaultFXFormValidator.java:61)
	at com.dooapp.fxform.AbstractFXForm.createFxFormValidatorProperty(AbstractFXForm.java:373)
	at com.dooapp.fxform.AbstractFXForm.fxFormValidatorProperty(AbstractFXForm.java:367)
	at com.dooapp.fxform.AbstractFXForm.createClassLevelValidator(AbstractFXForm.java:417)
	at com.dooapp.fxform.AbstractFXForm.getClassLevelValidator(AbstractFXForm.java:410)
	at com.dooapp.fxform.view.FXFormSkin.createClassLevelConstraintNode(FXFormSkin.java:240)
	at com.dooapp.fxform.view.skin.DefaultSkin.createRootNode(DefaultSkin.java:57)
	at com.dooapp.fxform.view.FXFormSkin.buildNode(FXFormSkin.java:101)
	at com.dooapp.fxform.view.FXFormSkin.<init>(FXFormSkin.java:96)
	at com.dooapp.fxform.view.skin.DefaultSkin.<init>(DefaultSkin.java:40)
	at com.dooapp.fxform.FXForm.<init>(FXForm.java:92)
	at com.dooapp.fxform.FXForm.<init>(FXForm.java:59)
	at com.dooapp.fxform.FXForm.<init>(FXForm.java:51)
	at com.dooapp.fxform.reflection.classcast.ClassCastTest.classCastTest(ClassCastTest.java:24)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.dooapp.fxform.JavaFXRule$1.evaluate(JavaFXRule.java:45)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

Reply:
It might be related to [that PR ](https://github.com/dooApp/FXForm2/pull/210/files)that was merged in between. I think you can just remove the el-api and jaxb-api now.

Reply:
https://docs.jboss.org/hibernate/validator/6.2/reference/en-US/html_single/#validator-gettingstarted-createproject

As define in hibernate-validor documentation the framework needs an EL implementation which was javax.el:el-api:2.2 with hibernate 5.1.
When upgrading to 6.2.x we now use org.glassfish:jakarta.el:3.0.3.

I suppose you had many warnings when you run the demo, that's why you added jakarta.el which was the right way to do. Thanks for that. So unless you have other warnings we didn't see jakarta.el must be the only dependency we should have in core.

Reply:
> It might be related to [that PR ](https://github.com/dooApp/FXForm2/pull/210/files)that was merged in between. I think you can just remove the el-api and jaxb-api now.

I did not mean to touch other parts. I just want upgrade javafx version. Perhaps, that was the problem.

Reply:
> I suppose you had many warnings when you run the demo, that's why you added jakarta.el which was the right way to do. Thanks for that. So unless you have other warnings we didn't see jakarta.el must be the only dependency we should have in core.

Yes.